proto/config.proto

```diff
@@ -94,3 +95,22 @@ message ZoneConfig {
   // in the zone config hierarchy, up to the default policy if necessary.
   optional GCPolicy gc = 4 [(gogoproto.customname) = "GC", (gogoproto.moretags) = "yaml:\"gc,omitempty\""];
 }
+
+// RangeTreeRoot holds the root node and size of the range tree.
+message RangeTree {
+  optional int64 root_id = 1 [(gogoproto.nullable) = true, (gogoproto.customname) = "RootID"];
+  optional int64 count = 2 [(gogoproto.nullable) = false, (gogoproto.customname) = "Count"];
```

Comment:
The `(gogoproto.custoname) = "Count"` option shouldn't be necessary. We generally only specify a custom name when the default naming that gogoproto would use does not match with standard Go naming. The most common case is `id` which gogoproto translates to `Id` while we want `ID`.